pages/_document.tsx

```diff
@@ -21,8 +25,14 @@ interface MyDocumentProps extends DocumentProps {
 }
 
 export default function MyDocument({ emotionStyleTags }: MyDocumentProps) {
+  const prefersDarkMode = useMediaQuery('(prefers-color-scheme: dark)');
```

Comment:
After implementing my previous comment, we will fix duplication of logic

Reply:
'+